internal/provider/provider.go

```diff
@@ -178,6 +178,7 @@ func (p *HumanitecProvider) Resources(ctx context.Context) []func() resource.Res
 func (p *HumanitecProvider) DataSources(ctx context.Context) []func() datasource.DataSource {
 	return []func() datasource.DataSource{
 		NewSourceIPRangesDataSource,
+		NewUsersDataSource,
```

Comment:
Could you also add a usage here https://github.com/humanitec/terraform-provider-humanitec/tree/main/examples/data-sources/?

Reply:
Done!